packages/form-js-viewer/src/render/hooks/useCondition.js

```diff
@@ -0,0 +1,16 @@
+import useService from './useService.js';
+
+/**
+ *
+ * @param {{ expression?: string } | undefined} condition
```

Comment:
What is the result of this hook?

Return type is missing; unclear if it returns "true" to show or to hide.